token/cli/src/main.rs

```diff
@@ -254,6 +255,21 @@ fn command_create_token(
             decimals,
         )?,
     ];
+
+    // check for memo text; if it exists convert it to
+    // bytes and add memo tx instructions
+    if let Some(text) = memo {
+        let memo_instruction = Instruction {
```

Comment:
Can we use `spl_memo::build_memo()` to save some code here? https://github.com/solana-labs/solana-program-library/blob/731bb6f57a676c2ad8c7a8ed2ea094f32d83cc5f/memo/program/src/lib.rs#L30